docs/DeveloperGuide.md

```diff
@@ -1,34 +1,503 @@
 # Developer Guide
 
-## Design & implementation
+![Trippie Logo](https://i.imgur.com/jSwGL7O.png)
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## Table of Content
+Below is a list of contents that is included in this document.
+* [1.0 Introduction](#10-Introduction)
+* [2.0 Setting up the project in your computer](#20-Setting-up-the-project-in-your-computer)
+* [3.0 Design](#30-Design)
+    * [3.1 Architecture](#31-Architecture-Felix)
+    * [3.2 Ui](#32-Ui-Felix)
+    * [3.3 Parser](#33-Parser-Wei-Shuang)
+    * [3.4 TrippieData](#34-TrippieData-Ivander)
+    * [3.5 Storage](#35-Storage-Ivander)
+    * [3.6 Command](#36-Command-Wei-Shuang)
+    * [3.7 Exceptions](#37-Exceptions-Wei-Shuang)
+* [4.0 Implementation](#40-Implementation-DG-to-complete-by-Wed-28-Oct-soft)
+    * [4.1 Multiple Trips Structure](#41-Multiple-Trips-Structure-Ivander)
+    * [4.2 Sorting Place List](#42-Sorting-Place-List-Kian-En)
+    * [4.3 Budget and Expenses](#43-Budget-and-Expenses-Shawn)
+    * [4.4 Foreign Exchange Converter](#Foreign-Exchange-Converter-Shawn)
+    * [4.5 Import and Export Files](#45-Import-and-Export-Files-Wei-Shuang)
+    * [4.6 [Proposed] Colors](#46-Proposed-Colors)
+* [5.0 Appendix: Requirements](#50-Appendix-Requirements)
+    * [5.1 Product Scope](#51-Product-scope)
+        * [5.1.0 Target user profile](#510-Target-user-profile)
+        * [5.1.1 Value proposition](#511-Value-proposition)
+    * [5.2 User Stories](#52-User-Stories)
+    * [5.3 Use Cases](#53-Use-Cases)
+    * [5.4 Non-Functional Requirements](#54-Non-Functional-Requirements)
+    * [5.5 Glossary](#55-Glossary)
+* [6.0 Appendix: Instructions for Manual Testing](#60-Appendix-Instructions-for-Manual-Testing)
+    * [6.1 Launch and Shutdown](#61-Launch-and-Shutdown-Felix)
```

Comment:
The table of contents hyperlinks do not send the user to the appropriate section. Are the hyperlinks formatted properly?